src/Model/Table/QueuedTasksTable.php

```diff
@@ -3,7 +3,7 @@
 
 use Cake\Core\Configure;
 use Cake\Utility\Hash;
-use Tools\Model\Table\Table;
+use Cake\ORM\Table;
```

Comment:
So, i need "cakephp-shim" installed to use the Queue?
I have replaced the shim parts with CakePHP default ones to keep my project small